app/api/questionsInLecture.js

```diff
@@ -0,0 +1,97 @@
+const router = require('express').Router({ mergeParams: true })
+const db = require('../models')
+const { generateUserAuthToken, requireAuthentication } = require('../../lib/auth')
+const enrollmentService = require('../services/enrollment_service')
+const lectureService = require('../services/lecture_service')
+const questionService = require('../services/question_service')
+
+// base path: /courses/:course_id/lectures/:lecture_id/questions
+
+// teacher wants to view a question inside a lecture 
```

Comment:
I haven't been following super closely the API design lately. Is this not the endpoint a student user would need as well?